crate_universe/src/metadata/dependency.rs

```diff
@@ -604,6 +604,23 @@ mod test {
         );
     }
 
+    #[test]
+    fn multi_kind_proc_macro_dep() {
```

Comment:
Can you also assert that the `build` dependency is correctly identified for build script targets?

Reply:
Done, thanks!